Airports/EGLC/SMR/Labels.txt

```diff
@@ -9,6 +9,7 @@
 "8" N051.30.15.850 E000.02.44.910 smrWhite
 "9" N051.30.15.920 E000.02.42.670 smrWhite
 "10" N051.30.15.990 E000.02.40.450 smrWhite
+"10L" N051.30.15.990 E000.02.40.450 smrWhite
```

Comment:
This is in exactly the same place as "10" - they don't display per your screenshot. Might be worth considering trying @hazzas-99 's suggestion on the origional issue of moving the "10" and just adding an "L" - depending how they look when viewing the whole aerodrome when moved a bit. This is what it looks like for me at the moment:
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/61326713/484884ef-667a-4ab1-a38b-0129ae40b77c)

Reply:
GITHUB is just being ehh to me, let me change it on here

Reply:
That's looking better but still some overlap for me:
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/61326713/681ecc4e-13b9-4fa2-bcdd-c29d14b3b327)

Reply:
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/60842647/e099fc4c-c93c-4345-8b0c-0495dfc28c5f)
think this should be okay , readable from where i am